src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -459,7 +459,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     //   scrolling event.
     // Arguments:
     // - mouseDelta: the mouse wheel delta that triggered this event.
-    void ControlInteractivity::_mouseTransparencyHandler(const double mouseDelta)
+    void ControlInteractivity::_mouseTransparencyHandler(const int32_t mouseDelta) const

```

Comment:
wat

Reply:
I had to change one of these due to the int -> float change.
When I did that I noticed that all of these 3 functions are only ever called by one single function which never passes a fractional number. So I changed the parameter type to the mouse-delta type we use in WinRT which is a `int32_t`.